src/page-modules/contact/components/input/description-modal.tsx

```diff
@@ -0,0 +1,88 @@
+import { Typo } from '@atb/components/typography';
+import style from './input.module.css';
+import { motion } from 'framer-motion';
+import { FocusScope } from '@react-aria/focus';
+import { MonoIcon } from '@atb/components/icon';
+import { useEffect, useRef } from 'react';
+import { Button } from '@atb/components/button';
+
+type DescriptionModalProps = {
+  title: string;
+  description: string;
+  ariaLabel: string;
+  isModalOpen: boolean;
+  closeModal: () => void;
+};
+
+const DescriptionModal = ({
+  title,
+  description,
+  ariaLabel,
+  isModalOpen,
+  closeModal,
+}: DescriptionModalProps) => {
+  const modalRef = useRef<HTMLDivElement | null>(null);
+
+  const handleEscapeOrClickOutside = (event: KeyboardEvent | MouseEvent) => {
+    if (event instanceof KeyboardEvent && event.key === 'Escape') {
+      closeModal();
+    } else if (
+      event instanceof MouseEvent &&
+      modalRef.current &&
+      !modalRef.current.contains(event.target as Node)
+    ) {
+      closeModal();
+    }
+  };
+
+  useEffect(() => {
+    document.addEventListener('keydown', handleEscapeOrClickOutside);
+    document.addEventListener('mousedown', handleEscapeOrClickOutside);
+
+    return () => {
+      document.removeEventListener('keydown', handleEscapeOrClickOutside);
+      document.removeEventListener('mousedown', handleEscapeOrClickOutside);
+    };
+  }, []);
+
+  if (!isModalOpen) return null;
+
+  return (
+    <motion.div
+      className={style.modal_wrapper}
+      initial={{ opacity: 0 }}
+      animate={{ opacity: 1 }}
+      exit={{ opacity: 0 }}
+      transition={{ ease: 'easeInOut', duration: 0.15 }}
+    >
+      <FocusScope contain restoreFocus autoFocus>
+        <motion.div
+          ref={modalRef}
+          initial={{ opacity: 0, y: -30 }}
+          animate={{ opacity: 1, y: 0 }}
+          exit={{ opacity: 0, y: 0 }}
+          transition={{ ease: 'backOut', delay: 0.1, duration: 0.15 }}
+        >
+          <dialog
+            aria-modal="true"
+            aria-label={ariaLabel}
+            aria-labelledby={`modal-${title}`}
```

Comment:
This should reference an element by ID (i.e. the h2 element inside the dialog in this case) (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby)